app/src/main/java/nu/yona/app/ui/settings/SettingsFragment.java

```diff
@@ -131,7 +135,8 @@ private View configureSettingsListView(View settingsLayoutView)
 				getString(R.string.adddevice),
 				getString(R.string.showopenvpnlog),
 				getString(R.string.contact_us),
-				getString(R.string.deleteuser)};
+				getString(R.string.deleteuser),
+				"STOP VPN", "START VPN"};
```

Comment:
Why is this different from the rest of the list? The strings are now repeated here and below, causing a risk that they grow apart.